Automattic/wp-calypso
# Plugins update manager: CRUD integration

Related to https://github.com/Automattic/dotcom-forge/issues/5544
Closes https://github.com/Automattic/dotcom-forge/issues/5646


## Proposed Changes

* Integrate fetching schedules endpoint
* Integrated create endpoint
* Integrated delete endpoint
* Fixed some small presentation bugs
* Shown spinner while fetching plugins and schedules
* Connected form validation with the existing scheduled timeslots and plugins

## Testing Instructions



* Go to `/plugins/scheduled-updates/{WoA_SITE_SLUG}`
* Check if create action with validation works properly
* Check if deleting scheduled updates works properly

**NOTE:** In the next PRs, busy state while the action is in progress will be covered

![Screen Capture on 2024-03-04 at 12-56-55](https://github.com/Automattic/wp-calypso/assets/1241413/c23ece51-c668-48a5-8d4b-b02388ff44d8)

![Screen Capture on 2024-02-29 at 18-32-13](https://github.com/Automattic/wp-calypso/assets/1241413/0f5abb7c-743c-4e32-b012-04dc78b31b86)
![Screen Capture on 2024-02-29 at 18-26-12](https://github.com/Automattic/wp-calypso/assets/1241413/ca002037-a159-4611-9bd2-6ee9c495acf7)


## Pre-merge Checklist
